src/box/box.cc

```diff
@@ -5963,18 +5963,20 @@ box_broadcast_id(void)
 	assert((size_t)(w - buf) < sizeof(buf));
 }
 
-static void
+void
 box_broadcast_status(void)
 {
 	char buf[1024];
 	char *w = buf;
-	w = mp_encode_map(w, 3);
+	w = mp_encode_map(w, 4);
 	w = mp_encode_str0(w, "is_ro");
 	w = mp_encode_bool(w, box_is_ro());
 	w = mp_encode_str0(w, "is_ro_cfg");
 	w = mp_encode_bool(w, cfg_geti("read_only"));
 	w = mp_encode_str0(w, "status");
 	w = mp_encode_str0(w, box_status());
+	w = mp_encode_str0(w, "schema_version");
+	w = mp_encode_str0(w, version_id_to_string(dd_version_id));
```

Comment:
Wouldn't it be better to return the version as an array, e.g. `{3, 1, 0}`? It'd be easier for comparisons.

Reply:
WDYT of returning `dd_version_id` itself, as unsigned integer? In the scope of https://github.com/tarantool/tarantool/issues/10521 `version` module will be introduced and I'm intending on adding `version.from_id(info.dd_version_id)` method there, so it'll be easy to use, even better IMHO than calling smth like `version.new(info.version[1], info.version[2], info.version[3])`.

P.S. RFC for version module is ready for review

Reply:
Once we add the version id to the public API, we won't be able to change its representation. In particular, using the version module as is would limit the max version number to 255:

https://github.com/tarantool/tarantool/blob/7cf41a1bb9b3f0a2eac82c296a764688f5d8b35c/src/box/lua/mkversion.lua#L42-L47

So I'd prefer returning an array or a string instead. If we export the version module, using strings would probably be fine.

Reply:
I think that a string or an array is fine. Can't say what is better -- so, I'm OK with any of these variants.

Reply:
I like string better, let's use it, since nobody cares